students/17th/hkk/hyeonsookang/user/views.py

```diff
@@ -0,0 +1,31 @@
+import json
+
+from django.http  import JsonResponse
+from django.views import View
+
+from user.models  import User
+
+class UserView(View):
+    def post(self, request):
+        try:
+            data             = json.loads(request.body)
+            passwordvalidity = len(data['password'])
+
+            if '@' not in data['email'] or '.' not in data['email']:
+                return JsonResponse({'MESSAGE': 'INVALID__EMAIL'}, status=401)
+
+            if User.objects.filter(email=data['email']).exists():
+                return JsonResponse({'MESSAGE': 'ALREADY_SIGNUP'}, status=401)
+
+            if passwordvalidity < 8:
+                return JsonResponse({'MESSAGE': 'INVALID_PASSWORD'}, status=401)
+
+            user = User.objects.create(
```

Comment:
그저 유저를 생성만 하면 되는데 재사용성 없는 변수를 선언하는 것은 메모리의 낭비입니다. 
```suggestion
            User.objects.create(
```